### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by FeatherTechnology** [community/community#116323] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

#### Hello Folks,

On last Saturday i have raised a ticket. But still now i didn't got any response from that ticket. My GitHub account has been suspended, im unable to work on my project . I'll attach the email that i received before. please anyone help me.

```
Hello @Ambroarun83,

We recently noticed that your GitHub account had a suspicious login. Out of an abundance of caution, we made the decision to force a password reset for the account associated with this email address.

We have no reason to believe that this was the result of a GitHub vulnerability. While we can’t speak to your particular case, in other similar account takeovers our users have told us that infostealer malware was discovered on their system.

Follow the link below to request a password reset token.

https://github.com/password_reset

```

**Comment by queenofcorgis** [comment, marked as answer]
Hi @FeatherTechnology, 

Thank you for your patience and for opening a ticket with our Support team. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
